tests/Dfe.PlanTech.Web.E2ETests/cypress/e2e/dynamic-page-validator/dpv-validators/check-answers-validator.js

```diff
@@ -0,0 +1,15 @@
+import { CleanText } from "../../../helpers/text-helpers";
+
+export const validateCheckAnswersPage = (path, section) => {
+
+    for (const q of path) {
+        cy.get("div.govuk-summary-list__row dt.govuk-summary-list__key.spacer")
+            .contains(CleanText(q.question.text))
+            .siblings("dd.govuk-summary-list__value.spacer")
+            .contains(CleanText(q.answer.text));
+    }
+
+    cy.url().should("include", `${section.interstitialPage.fields.slug}/check-answers`);
+
+    cy.get("button.govuk-button").contains("Save and continue").click();
```

Comment:
What is the purpose of the `.click()" here? We're not validating anything after. It might be best to remove that, or to add a test after that we're now on the expected page (i.e. self-assessment page?).

Reply:
This is the click on the check answers page that submits the assessment, which takes us back to the self assessment page. I'll reorder a bit though so the url check happens before the Q&A checks because it makes a bit more sense that way.